### PR TITLE
fix typos in docs

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v8.1 (TiDB 8.1 versions)
- [x] v8.0 (TiDB 8.0 versions)
- [x] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s): https://github.com/pingcap/docs-cn/pull/17247

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: releases/release-6.1.1.md [hunk 1]
```diff
@@ -81,7 +81,7 @@ Quick access: [Quick start](https://docs.pingcap.com/tidb/v6.1/quick-start-with-
     - Fix the issue that the wrong join reorder in some right outer join scenarios causes wrong query result [#36912](https://github.com/pingcap/tidb/issues/36912) @[winoros](https://github.com/winoros)
     - Fix the issue of incorrectly inferred null flag of the TiFlash `firstrow` aggregate function in the EqualAll case [#34584](https://github.com/pingcap/tidb/issues/34584) @[fixdb](https://github.com/fixdb)
     - Fix the issue that Plan Cache does not work when a binding is created with the `IGNORE_PLAN_CACHE` hint [#34596](https://github.com/pingcap/tidb/issues/34596) @[fzzf678](https://github.com/fzzf678)
-    - Fix the issu that an `EXCHANGE` operator is missing between the hash-partition window and the single-partition window [#35990](https://github.com/pingcap/tidb/issues/35990) @[LittleFall](https://github.com/LittleFall)
+    - Fix the issue that an `EXCHANGE` operator is missing between the hash-partition window and the single-partition window [#35990](https://github.com/pingcap/tidb/issuees/35990) @[LittleFall](https://github.com/LittleFall)
```
Comment (qiancai) [inline releases/release-6.1.1.md:84]:
```suggestion
    - Fix the issue that an `EXCHANGE` operator is missing between the hash-partition window and the single-partition window [#35990](https://github.com/pingcap/tidb/issues/35990) @[LittleFall](https://github.com/LittleFall)
```

File: sql-statements/sql-statement-show-indexes.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ The statement `SHOW INDEXES [FROM|IN]` lists the indexes on a specified table. T
 
 ```ebnf+diagram
 ShowIndexStmt ::=
-    "SHOW" ( "INDEX" | "INDEXES" | "KEYS" ) ("FROM" | "IN" ) TableName (("FROM" | "IN") SchemaName )? ShowLikeOrWhere?
+    "SHOW" ( "INDEX" | "INDEXES" | "KEYS" ) ("FROM" | "IN" ) TableName (("FROM" | "IN") SchemaName )? ShowLikeOrWhereOpt?
```
Comment (qiancai) [inline sql-statements/sql-statement-show-indexes.md:15]:
```suggestion
    "SHOW" ( "INDEX" | "INDEXES" | "KEYS" ) ("FROM" | "IN" ) TableName (("FROM" | "IN") SchemaName )? ShowLikeOrWhere?
```

File: tikv-configuration-file.md [hunk 3]
```diff
@@ -1325,7 +1325,7 @@ Configuration items related to RocksDB
 
 + Unit: KiB|MiB|GiB
 
-### `track-and-verify-wals-in-manifest` <span class="version-mark">New in v6.5.9, v7.1.5, and v8.0.0</span>
+### `track-and-verify-wals-in-manifest` <span class="version-mark">New in v6.5.9 and v8.0.0</span>
```
Comment (qiancai) [inline tikv-configuration-file.md:1328]:
```suggestion
### `track-and-verify-wals-in-manifest` <span class="version-mark">New in v6.5.9, v7.1.5, and v8.0.0</span>
```

File: troubleshoot-lock-conflicts.md [hunk 4]
```diff
@@ -263,7 +263,7 @@ You can check whether there is any "LockNotFound" error in the following ways:
     If a "TxnLockNotFound" error occurs, the TiKV log message is like this:
 
     ```log
-    Error: KV error safe to retry restarts txn: Txn(Mvcc(TxnLockNotFound)) [ERROR [Kv.rs:708] ["KvService::batch_raft send response fail"] [err=RemoteStoped]
+    Error: KV error safe to retry restarts txn: Txn(Mvcc(TxnLockNotFound)) [ERROR [Kv.rs:708] ["KvService::batch_raft send response fail"] [err=RemoteStopped]
```
Comment (qiancai) [inline troubleshoot-lock-conflicts.md:266]:
```suggestion
    Error: KV error safe to retry restarts txn: Txn(Mvcc(TxnLockNotFound)) [ERROR [Kv.rs:708] ["KvService::batch_raft send response fail"] [err=RemoteStoped]
```

File: dm/dm-open-api.md [hunk 5]
```diff
@@ -1346,7 +1346,7 @@ curl -X 'GET' \
       "name": "string",
       "source_name": "string",
       "worker_name": "string",
-      "stage": "runing",
+      "stage": "running",
```
Comment (qiancai) [inline dm/dm-open-api.md:1349]: TBD

File: dashboard/dashboard-diagnostics-report.md [hunk 6]
```diff
@@ -266,7 +266,7 @@ Other monitoring tables in TiDB are as follows:
 The tables related to the monitoring information of PD modules are as follows:
 
 * `Time Consumed by PD Component`: The time consumed by the monitoring metrics of related modules in PD.
-* `Blance Leader/Region`: The monitoring information of `balance-region` and `balance leader` occurred in the cluster within the report time range, such as the number of leaders that are scheduled out from `tikv_note_1` or the number of leaders that are scheduled in.
+* `Balance Leader/Region`: The monitoring information of `balance-region` and `balance leader` occurred in the cluster within the report time range, such as the number of leaders that are scheduled out from `tikv_note_1` or the number of leaders that are scheduled in.
```
Comment (qiancai) [inline dashboard/dashboard-diagnostics-report.md:269]: TBD

File: tiup/tiup-mirror-reference.md [hunk 7]
```diff
@@ -154,7 +154,7 @@ The index file's format is as follows:
                     }
                 },
                 "name": "{owner-name}",                         # The name of the owner.
-                "threshod": {N}                                 # Indicates that the components owned by the owner must have at least N valid signatures.
+                "threshold": {N}                                 # Indicates that the components owned by the owner must have at least N valid signatures.
```
Comment (qiancai) [inline tiup/tiup-mirror-reference.md:157]: TBD

File: br/br-monitoring-and-alert.md [hunk 8]
```diff
@@ -25,7 +25,7 @@ Log backup supports using [Prometheus](https://prometheus.io/) to collect monito
 
 | Metrics                                                | Type    |  Description                                                                                                                                                 |
 |-------------------------------------------------------|-----------|---------------------------------------------------------------------------------------------------------------------------------------------------------|
-| **tikv_log_backup_interal_actor_acting_duration_sec** | Histogram | The duration of handling all internal messages and events. <br/>`message :: TaskType`                                                                                                            |
+| **tikv_log_backup_internal_actor_acting_duration_sec** | Histogram | The duration of handling all internal messages and events. <br/>`message :: TaskType`                                                                                                            |
```
Comment (qiancai) [inline br/br-monitoring-and-alert.md:28]:
```suggestion
| **tikv_log_backup_interal_actor_acting_duration_sec** | Histogram | The duration of handling all internal messages and events. <br/>`message :: TaskType`                                                                                                            |
```

File: dashboard/dashboard-diagnostics-report.md (hunk 6 above)
Comment (qiancai) [inline dashboard/dashboard-diagnostics-report.md:269]:
```suggestion
* `Blance Leader/Region`: The monitoring information of `balance-region` and `balance leader` occurred in the cluster within the report time range, such as the number of leaders that are scheduled out from `tikv_note_1` or the number of leaders that are scheduled in.
```

**Review by hfxsd — APPROVED** [review]

**Pull request merged**